code/game/objects/items/tools/shovel_tools.dm

```diff
@@ -23,6 +23,8 @@
 	var/dirt_amt = 0
 	var/dirt_amt_per_dig = 5
 
+	var/entrenchingmode = FALSE //Toggles if the shovel will dig out dirt or a trench
```

Comment:
This is not how you should comment

Do this :
/// Toggles if the shovel will dig out dirt or a trench (yes, triple ///)
var/entrechingmode = FALSE